### PR TITLE
Standardizing the mount module

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Sending a separate pull request for the mount module

File: library/mount [hunk 1]
```diff
@@ -166,10 +150,11 @@ def mount(**kwargs):
     if call.returncode == 0:
         return 0, ''
     else:
-        return call.rc, out+err
```
Comment (ghost) [inline library/mount:169]: call.rc fails. fixed that

File: library/mount [hunk 2]
```diff
@@ -166,10 +150,11 @@ def mount(**kwargs):
...
+        return call.returncode, out+err
 
 def umount(**kwargs):
     "unmount a path"
+    name = kwargs['name']
```
Comment (ghost) [inline library/mount:157]: name was not initialized earlier. the module failed because of that. fixed that too

**Comment by mpdehaan** [comment]
Looks good, thanks!

**Pull request merged**